cdap-data-fabric-tests/src/test/java/co/cask/cdap/data2/transaction/stream/hbase/HBaseConsumerStateTest.java

```diff
@@ -89,7 +91,13 @@ public static void init() throws Exception {
       new LocationRuntimeModule().getInMemoryModules(),
       new DiscoveryRuntimeModule().getInMemoryModules(),
       new TransactionMetricsModule(),
-      new DataFabricDistributedModule(),
+      Modules.override(new DataFabricDistributedModule())
+      .with(new AbstractModule() {
+        @Override
+        protected void configure() {
+          bind(NamespaceQueryAdmin.class).to(SimpleNamespaceQueryAdmin.class);
```

Comment:
This overriding is done at multiple places in this PR. But I don't see that ``DataFabricDistributedModule`` has a binding to ``NamespaceQueryAdmin``. Why is this necessary?